acceptance/tests/provider/package/dpkg_ensure_held_package_is_latest.rb

```diff
@@ -14,13 +14,11 @@
 end
 
 step"Ensure that package is installed first if not present" do
-  info = on(agent.name, "apt-cache policy #{package} | grep Candidate:").stdout
-  expected_package_version = info.split.first
+  expected_package_version = on(agent.name, "apt-cache policy #{package} | sed -n -e 's/Candidate: //p'").stdout
```

Comment:
@loopinu I missed this when the test was originally created, but can you wrap this entire block in:

```ruby
agents.each do |host|
  ... = on(host.name, ...)
end
```

As it is now, the test will explode if there are no host with the `agent` role, or if there are multiple hosts, due to some odd beaker behavior.